--- template/app/src/file-upload/file-upload.wasp.ts
+++ opensaas-sh/app/src/file-upload/file-upload.wasp.ts
@@ -1,4 +1,4 @@
-import { action, page, query, route, type Spec } from "@wasp.sh/spec";
+import { action, job, page, query, route, type Spec } from "@wasp.sh/spec";
 
 import { FileUploadPage } from "./FileUploadPage" with { type: "ref" };
 import {
@@ -8,6 +8,7 @@
   getAllFilesByUser,
   getDownloadFileSignedURL,
 } from "./operations" with { type: "ref" };
+import { deleteFilesJob } from "./workers" with { type: "ref" };
 
 export const fileUploadSpec: Spec = [
   route(
@@ -20,4 +21,11 @@
   action(addFileToDb, { entities: ["User", "File"] }),
   action(createFileUploadUrl, { entities: ["User", "File"] }),
   action(deleteFile, { entities: ["User", "File"] }),
+  job(deleteFilesJob, {
+    executor: "PgBoss",
+    schedule: {
+      cron: "0 5 * * *", // every day at 5am
+    },
+    entities: ["File"],
+  }),
 ];
