diff --git a/main.wasp.ts b/main.wasp.ts
index e3191d4..64b489a 100644
--- a/main.wasp.ts
+++ b/main.wasp.ts
@@ -1,5 +1,6 @@
 import { MainPage } from "./src/MainPage" with { type: "ref" }
-import { app, page, route } from "@wasp.sh/spec"
+import { getTasks } from "./src/queries" with { type: "ref" }
+import { app, page, query, route } from "@wasp.sh/spec"
 
 export default app({
   name: "TodoApp",
@@ -8,5 +9,6 @@ export default app({
   head: ["<link rel='icon' href='/favicon.ico' />"],
   parts: [
     route("RootRoute", "/", page(MainPage)),
+    query(getTasks, { entities: ["Task"] }),
   ],
 });
