diff --git a/main.wasp.ts b/main.wasp.ts
index 11f129d..2f7bbe9 100644
--- a/main.wasp.ts
+++ b/main.wasp.ts
@@ -1,5 +1,6 @@
-import { app, page, route } from "@wasp.sh/spec";
+import { app, page, query, route } from "@wasp.sh/spec";
 import { MainPage } from "./src/MainPage" with { type: "ref" };
+import { getTasks } from "./src/queries" with { type: "ref" };
 
 export default app({
   name: "TodoApp",
@@ -11,5 +12,6 @@ export default app({
   },
   spec: [
     route("RootRoute", "/", page(MainPage)),
+    query(getTasks, { entities: ["Task"] }),
   ],
 });
