diff --git a/main.wasp.tsx b/main.wasp.tsx
index dc1b39f..53e10fc 100644
--- a/main.wasp.tsx
+++ b/main.wasp.tsx
@@ -1,5 +1,6 @@
-import { app, page, route } from "@wasp.sh/spec";
+import { app, page, query, route } from "@wasp.sh/spec";
 import { MainPage } from "./src/MainPage" with { type: "ref" };
+import { getTasks } from "./src/queries" with { type: "ref" };
 
 export default app({
   name: "TodoApp",
@@ -8,5 +9,6 @@ export default app({
   head: [<link rel="icon" href="/favicon.ico" />],
   spec: [
     route("RootRoute", "/", page(MainPage)),
+    query(getTasks, { entities: ["Task"] }),
   ],
 });
