diff --git a/main.wasp.ts b/main.wasp.ts
index b0c2589..64d2908 100644
--- a/main.wasp.ts
+++ b/main.wasp.ts
@@ -1,4 +1,4 @@
-import { createTask } from "./src/actions" with { type: "ref" }
+import { createTask, updateTask } from "./src/actions" with { type: "ref" }
 import { MainPage } from "./src/MainPage" with { type: "ref" }
 import { getTasks } from "./src/queries" with { type: "ref" }
 import { action, app, page, query, route } from "@wasp.sh/spec"
@@ -12,5 +12,6 @@ export default app({
     route("RootRoute", "/", page(MainPage)),
     query(getTasks, { entities: ["Task"] }),
     action(createTask, { entities: ["Task"] }),
+    action(updateTask, { entities: ["Task"] }),
   ],
 });
