diff --git a/main.wasp.ts b/main.wasp.ts
index 53e10fc..b4e49b4 100644
--- a/main.wasp.ts
+++ b/main.wasp.ts
@@ -1,6 +1,7 @@
-import { app, page, query, route } from "@wasp.sh/spec";
+import { action, app, page, query, route } from "@wasp.sh/spec";
 import { MainPage } from "./src/MainPage" with { type: "ref" };
 import { getTasks } from "./src/queries" with { type: "ref" };
+import { createTask } from "./src/actions" with { type: "ref" };
 
 export default app({
   name: "TodoApp",
@@ -10,5 +11,6 @@ export default app({
   spec: [
     route("RootRoute", "/", page(MainPage)),
     query(getTasks, { entities: ["Task"] }),
+    action(createTask, { entities: ["Task"] }),
   ],
 });
