diff --git a/main.wasp.ts b/main.wasp.ts
index 4246d1a..5323589 100644
--- a/main.wasp.ts
+++ b/main.wasp.ts
@@ -8,6 +8,13 @@ export default app({
   wasp: { version: "^0.25.0" },
   title: "TodoApp",
   head: ["<link rel='icon' href='/favicon.ico' />"],
+  auth: {
+    userEntity: "User",
+    methods: {
+      usernameAndPassword: {},
+    },
+    onAuthFailedRedirectTo: "/login",
+  },
   server: {
     basePath: "/api",
   },
