diff --git a/main.wasp.ts b/main.wasp.ts
index f6d0dfc..1d820cf 100644
--- a/main.wasp.ts
+++ b/main.wasp.ts
@@ -1,4 +1,6 @@
 import { createTask, updateTask } from "./src/actions" with { type: "ref" }
+import { SignupPage } from "./src/SignupPage" with { type: "ref" }
+import { LoginPage } from "./src/LoginPage" with { type: "ref" }
 import { MainPage } from "./src/MainPage" with { type: "ref" }
 import { getTasks } from "./src/queries" with { type: "ref" }
 import { action, app, page, query, route } from "@wasp.sh/spec"
@@ -20,5 +22,7 @@ export default app({
     query(getTasks, { entities: ["Task"] }),
     action(createTask, { entities: ["Task"] }),
     action(updateTask, { entities: ["Task"] }),
+    route("SignupRoute", "/signup", page(SignupPage)),
+    route("LoginRoute", "/login", page(LoginPage)),
   ],
 });
