diff --git a/main.wasp.tsx b/main.wasp.tsx
index 894c99a..52438c9 100644
--- a/main.wasp.tsx
+++ b/main.wasp.tsx
@@ -1,5 +1,7 @@
 import { action, app, page, query, route } from "@wasp.sh/spec";
 import { MainPage } from "./src/MainPage" with { type: "ref" };
+import { SignupPage } from "./src/SignupPage" with { type: "ref" };
+import { LoginPage } from "./src/LoginPage" with { type: "ref" };
 import { getTasks } from "./src/queries" with { type: "ref" };
 import { createTask, updateTask } from "./src/actions" with { type: "ref" };
 
@@ -17,6 +19,8 @@ export default app({
   },
   spec: [
     route("RootRoute", "/", page(MainPage)),
+    route("SignupRoute", "/signup", page(SignupPage)),
+    route("LoginRoute", "/login", page(LoginPage)),
     query(getTasks, { entities: ["Task"] }),
     action(createTask, { entities: ["Task"] }),
     action(updateTask, { entities: ["Task"] }),
