diff --git a/main.wasp.tsx b/main.wasp.tsx
index 52438c9..6a71803 100644
--- a/main.wasp.tsx
+++ b/main.wasp.tsx
@@ -18,7 +18,13 @@ export default app({
     onAuthFailedRedirectTo: "/login",
   },
   spec: [
-    route("RootRoute", "/", page(MainPage)),
+    route(
+      "RootRoute",
+      "/",
+      page(MainPage, {
+        authRequired: true,
+      }),
+    ),
     route("SignupRoute", "/signup", page(SignupPage)),
     route("LoginRoute", "/login", page(LoginPage)),
     query(getTasks, { entities: ["Task"] }),
