diff --git a/main.wasp b/main.wasp
index f761afe..d4798a4 100644
--- a/main.wasp
+++ b/main.wasp
@@ -5,7 +5,17 @@ app TodoApp {
   title: "TodoApp",
   head: [
     "<link rel='icon' href='/favicon.ico' />",
-  ]
+  ],
+  auth: {
+    // Tells Wasp which entity to use for storing users.
+    userEntity: User,
+    methods: {
+      // Enable username and password auth.
+      usernameAndPassword: {}
+    },
+    // We'll see how this is used in a bit.
+    onAuthFailedRedirectTo: "/login"
+  }
 }
 
 route RootRoute { path: "/", to: MainPage }
