diff --git a/main.wasp b/main.wasp
index eed193e..5592563 100644
--- a/main.wasp
+++ b/main.wasp
@@ -2,7 +2,17 @@ app TodoApp {
   wasp: {
     version: "^0.17.0"
   },
-  title: "TodoApp"
+  title: "TodoApp",
+  auth: {
+    // Tells Wasp which entity to use for storing users.
+    userEntity: User,
+    methods: {
+      // Enable username and password auth.
+      usernameAndPassword: {}
+    },
+    // We'll see how this is used in a bit.
+    onAuthFailedRedirectTo: "/login"
+  }
 }
 
 route RootRoute { path: "/", to: MainPage }
