diff --git a/main.wasp.ts b/main.wasp.ts
index 9c8106a..cf512c7 100644
--- a/main.wasp.ts
+++ b/main.wasp.ts
@@ -21,7 +21,13 @@ export default app({
     basePath: "/api",
   },
   spec: [
-    route("RootRoute", "/", page(MainPage)),
+    route(
+      "RootRoute",
+      "/",
+      page(MainPage, {
+        authRequired: true,
+      }),
+    ),
     route("SignupRoute", "/signup", page(SignupPage)),
     route("LoginRoute", "/login", page(LoginPage)),
     query(getTasks, { entities: ["Task"] }),
