diff --git a/main.wasp.ts b/main.wasp.ts
index 4d3c796..cfdf5eb 100644
--- a/main.wasp.ts
+++ b/main.wasp.ts
@@ -8,6 +8,13 @@ export default app({
   wasp: { version: "^0.24.0" },
   title: "TodoApp",
   head: ["<link rel='icon' href='/favicon.ico' />"],
+  auth: {
+    userEntity: "User",
+    methods: {
+      usernameAndPassword: {},
+    },
+    onAuthFailedRedirectTo: "/login",
+  },
   spec: [
     route("RootRoute", "/", page(MainPage)),
     query(getTasks, { entities: ["Task"] }),
