diff --git a/main.wasp.ts b/main.wasp.ts
index 4d3c796..c8b8c83 100644
--- a/main.wasp.ts
+++ b/main.wasp.ts
@@ -1,5 +1,7 @@
 import { action, app, page, query, route } from "@wasp.sh/spec";
 import { MainPage } from "./src/MainPage" with { type: "ref" };
+import { SignupPage } from "./src/SignupPage" with { type: "ref" };
+import { LoginPage } from "./src/LoginPage" with { type: "ref" };
 import { getTasks } from "./src/queries" with { type: "ref" };
 import { createTask, updateTask } from "./src/actions" with { type: "ref" };
 
@@ -8,8 +10,16 @@ export default app({
   wasp: { version: "^0.24.0" },
   title: "TodoApp",
   head: ["<link rel='icon' href='/favicon.ico' />"],
+  auth: {
+    userEntity: "User",
+    methods: {
+      usernameAndPassword: {},
+    },
+    onAuthFailedRedirectTo: route("LoginRoute", "/login", page(LoginPage)),
+  },
   spec: [
     route("RootRoute", "/", page(MainPage)),
+    route("SignupRoute", "/signup", page(SignupPage)),
     query(getTasks, { entities: ["Task"] }),
     action(createTask, { entities: ["Task"] }),
     action(updateTask, { entities: ["Task"] }),
