diff --git a/main.wasp b/main.wasp
index 6b4593c..c3df1d1 100644
--- a/main.wasp
+++ b/main.wasp
@@ -8,4 +8,14 @@ app TodoApp {
 route RootRoute { path: "/", to: MainPage }
 page MainPage {
   component: import { MainPage } from "@src/MainPage"
-}
\ No newline at end of file
+}
+
+query getTasks {
+  // Specifies where the implementation for the query function is.
+  // The path `@src/queries` resolves to `src/queries.ts`.
+  // No need to specify an extension.
+  fn: import { getTasks } from "@src/queries",
+  // Tell Wasp that this query reads from the `Task` entity. Wasp will
+  // automatically update the results of this query when tasks are modified.
+  entities: [Task]
+}
