diff --git a/src/MainPage.tsx b/src/MainPage.tsx
index 556e1a2..50118d6 100644
--- a/src/MainPage.tsx
+++ b/src/MainPage.tsx
@@ -1,6 +1,11 @@
+import type { FormEvent } from 'react'
 import type { Task } from 'wasp/entities'
-// highlight-next-line
-import { getTasks, useQuery } from 'wasp/client/operations'
+import {
+  // highlight-next-line
+  createTask,
+  getTasks,
+  useQuery,
+} from 'wasp/client/operations'
 
 export const MainPage = () => {
   // highlight-start
@@ -38,4 +43,27 @@ const TasksList = ({ tasks }: { tasks: Task[] }) => {
     </div>
   )
 }
-// highlight-end
\ No newline at end of file
+// highlight-end
+
+// highlight-start
+const NewTaskForm = () => {
+  const handleSubmit = async (event: FormEvent<HTMLFormElement>) => {
+    event.preventDefault()
+    try {
+      const target = event.target as HTMLFormElement
+      const description = target.description.value
+      target.reset()
+      await createTask({ description })
+    } catch (err: any) {
+      window.alert('Error: ' + err.message)
+    }
+  }
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <input name="description" type="text" defaultValue="" />
+      <input type="submit" value="Create task" />
+    </form>
+  )
+}
+// highlight-end
