diff --git a/src/MainPage.tsx b/src/MainPage.tsx
index 6075b9b..b1f94f5 100644
--- a/src/MainPage.tsx
+++ b/src/MainPage.tsx
@@ -8,6 +8,7 @@ import {
   useQuery,
 } from 'wasp/client/operations'
 import { AuthUser } from 'wasp/auth'
+import { logout } from 'wasp/client/auth'
 
 export const MainPage = ({ user }: { user: AuthUser }) => {
   // highlight-start
@@ -20,6 +21,7 @@ export const MainPage = ({ user }: { user: AuthUser }) => {
 
       {isLoading && 'Loading...'}
       {error && 'Error: ' + error}
+      <button onClick={logout}>Logout</button>
     </div>
   )
   // highlight-end
