diff --git a/src/actions.ts b/src/actions.ts
index 3edb8fb..45c82eb 100644
--- a/src/actions.ts
+++ b/src/actions.ts
@@ -1,5 +1,5 @@
 import { Task } from 'wasp/entities'
-import { CreateTask } from 'wasp/server/operations'
+import { CreateTask, UpdateTask } from 'wasp/server/operations'
 
 type CreateTaskPayload = Pick<Task, 'description'>
 
@@ -10,4 +10,18 @@ export const createTask: CreateTask<CreateTaskPayload, Task> = async (
   return context.entities.Task.create({
     data: { description: args.description },
   })
-}
\ No newline at end of file
+}
+
+type UpdateTaskPayload = Pick<Task, 'id' | 'isDone'>
+
+export const updateTask: UpdateTask<UpdateTaskPayload, Task> = async (
+  { id, isDone },
+  context
+) => {
+  return context.entities.Task.update({
+    where: { id },
+    data: {
+      isDone: isDone,
+    },
+  })
+}
