diff --git a/src/MainPage.tsx b/src/MainPage.tsx
index f02e05e..a4389aa 100644
--- a/src/MainPage.tsx
+++ b/src/MainPage.tsx
@@ -7,8 +7,9 @@ import {
   getTasks,
   useQuery,
 } from 'wasp/client/operations'
+import { AuthUser } from 'wasp/auth'
 
-export const MainPage = () => {
+export const MainPage = ({ user }: { user: AuthUser }) => {
   // highlight-start
   const { data: tasks, isLoading, error } = useQuery(getTasks)
 
