--- template/app/src/client/components/ui/button.tsx
+++ opensaas-sh/app/src/client/components/ui/button.tsx
@@ -10,21 +10,26 @@
     variants: {
       variant: {
         default:
-          "bg-primary text-primary-foreground shadow hover:bg-primary/90",
+          "bg-primary text-primary-foreground shadow-outer hover:bg-primary/90",
         destructive:
           "bg-destructive text-destructive-foreground shadow-sm hover:bg-destructive/90",
         outline:
           "border border-input bg-background shadow-sm hover:bg-accent hover:text-accent-foreground",
         secondary:
           "bg-secondary text-secondary-foreground shadow-sm hover:bg-secondary/80",
-        ghost: "hover:bg-accent hover:text-accent-foreground",
+        ghost: "hover:bg-muted hover:text-foreground text-muted-foreground",
         link: "text-primary underline-offset-4 hover:underline",
+        selected: "border bg-muted text-muted-foreground",
+        outer: "shadow-outer bg-card text-card-foreground",
+        inner:
+          "shadow-inner bg-secondary-muted text-secondary-muted-foreground",
       },
       size: {
         default: "h-9 px-4 py-2",
         sm: "h-8 rounded-md px-3 text-xs",
         lg: "h-10 rounded-md px-8",
         icon: "h-9 w-9",
+        iconLg: "h-12 w-12",
       },
     },
     defaultVariants: {
