--- template/app/src/auth/userSignupFields.ts
+++ opensaas-sh/app/src/auth/userSignupFields.ts
@@ -1,11 +1,6 @@
 import { defineUserSignupFields } from "wasp/auth/providers/types";
-import { env } from "wasp/server";
 import { z } from "zod";
 
-function isAdminEmail(email: string): boolean {
-  return env.ADMIN_EMAILS.includes(email);
-}
-
 const emailDataSchema = z.object({
   email: z.string(),
 });
@@ -19,10 +14,6 @@
     const emailData = emailDataSchema.parse(data);
     return emailData.email;
   },
-  isAdmin: (data) => {
-    const emailData = emailDataSchema.parse(data);
-    return isAdminEmail(emailData.email);
-  },
 });
 
 const githubDataSchema = z.object({
@@ -51,14 +42,6 @@
     const githubData = githubDataSchema.parse(data);
     return githubData.profile.login;
   },
-  isAdmin: (data) => {
-    const githubData = githubDataSchema.parse(data);
-    const emailInfo = getGithubEmailInfo(githubData);
-    if (!emailInfo.verified) {
-      return false;
-    }
-    return isAdminEmail(emailInfo.email);
-  },
 });
 
 // We are using the first email from the list of emails returned by GitHub.
@@ -91,13 +74,6 @@
     const googleData = googleDataSchema.parse(data);
     return googleData.profile.email;
   },
-  isAdmin: (data) => {
-    const googleData = googleDataSchema.parse(data);
-    if (!googleData.profile.email_verified) {
-      return false;
-    }
-    return isAdminEmail(googleData.profile.email);
-  },
 });
 
 export function getGoogleAuthConfig() {
@@ -129,13 +105,6 @@
     const discordData = discordDataSchema.parse(data);
     return discordData.profile.username;
   },
-  isAdmin: (data) => {
-    const discordData = discordDataSchema.parse(data);
-    if (!discordData.profile.email || !discordData.profile.verified) {
-      return false;
-    }
-    return isAdminEmail(discordData.profile.email);
-  },
 });
 
 export function getDiscordAuthConfig() {
