diff --git a/src/actions.ts b/src/actions.ts
index b8b2026..973d821 100644
--- a/src/actions.ts
+++ b/src/actions.ts
@@ -1,5 +1,5 @@
 import type { Task } from "wasp/entities";
-import type { CreateTask } from "wasp/server/operations";
+import type { CreateTask, UpdateTask } from "wasp/server/operations";
 
 type CreateTaskPayload = Pick<Task, "description">;
 
@@ -11,3 +11,17 @@ export const createTask: CreateTask<CreateTaskPayload, Task> = async (
     data: { description: args.description },
   });
 };
+
+type UpdateTaskPayload = Pick<Task, "id" | "isDone">;
+
+export const updateTask: UpdateTask<UpdateTaskPayload, Task> = async (
+  { id, isDone },
+  context,
+) => {
+  return context.entities.Task.update({
+    where: { id },
+    data: {
+      isDone: isDone,
+    },
+  });
+};
