--- template/app/migrations/20250806121259_add_s3_key_file/migration.sql
+++ opensaas-sh/app/migrations/20250806121259_add_s3_key_file/migration.sql
@@ -0,0 +1,11 @@
+/*
+  Warnings:
+
+  - You are about to drop the column `key` on the `File` table. All the data in the column will be lost.
+  - Added the required column `s3Key` to the `File` table without a default value. This is not possible if the table is not empty.
+
+*/
+-- AlterTable
+DELETE FROM "File";
+ALTER TABLE "File" DROP COLUMN "key",
+ADD COLUMN     "s3Key" TEXT NOT NULL;
