--- template/app/src/auth/email-and-pass/emails.ts
+++ opensaas-sh/app/src/auth/email-and-pass/emails.ts
@@ -6,10 +6,10 @@
 export const getVerificationEmailContent: GetVerificationEmailContentFn = ({
   verificationLink,
 }) => ({
-  subject: "Verify your email",
-  text: `Click the link below to verify your email: ${verificationLink}`,
+  subject: "Open SaaS - Verify your email",
+  text: `Open SaaS - Click the link to verify your email: ${verificationLink}`,
   html: `
-        <p>Click the link below to verify your email</p>
+        <p>Open SaaS - Click the link to verify your email:</p>
         <a href="${verificationLink}">Verify email</a>
     `,
 });
@@ -17,10 +17,10 @@
 export const getPasswordResetEmailContent: GetPasswordResetEmailContentFn = ({
   passwordResetLink,
 }) => ({
-  subject: "Password reset",
-  text: `Click the link below to reset your password: ${passwordResetLink}`,
+  subject: "Open SaaS - Password reset",
+  text: `Open SaaS - Click the link to reset your password: ${passwordResetLink}`,
   html: `
-        <p>Click the link below to reset your password</p>
+        <p>Open SaaS - Click the link to reset your password:</p>
         <a href="${passwordResetLink}">Reset password</a>
     `,
 });
