--- template/app/src/server/emailSender.wasp.ts
+++ opensaas-sh/app/src/server/emailSender.wasp.ts
@@ -4,10 +4,10 @@
   // NOTE: "Dummy" provider is just for local development purposes.
   //   Make sure to check the server logs for the email confirmation url (it will not be sent to an address)!
   //   Once you are ready for production, switch to e.g. "SendGrid" or "Mailgun" providers. Check out https://docs.opensaas.sh/guides/email-sending/ .
-  provider: "Dummy",
+  provider: "Mailgun",
   defaultFrom: {
     name: "Open SaaS App",
     // When using a real provider, e.g. SendGrid, you must use the same email address that you configured your account to send out emails with!
-    email: "me@example.com",
+    email: "noreply@mg.wasp.sh",
   },
 };
