--- template/app/src/landing-page/logos/ReactLogo.tsx
+++ opensaas-sh/app/src/landing-page/logos/ReactLogo.tsx
@@ -0,0 +1,28 @@
+export function ReactLogo() {
+  return (
+    <svg
+      width={32}
+      height={32}
+      xmlns="http://www.w3.org/2000/svg"
+      viewBox="-11.5 -10.23174 23 20.46348"
+    >
+      <circle
+        cx="0"
+        cy="0"
+        r="2.05"
+        fill="currentColor"
+        className="dark:fill-white"
+      />
+      <g
+        stroke="currentColor"
+        strokeWidth="1"
+        fill="none"
+        className="dark:stroke-white"
+      >
+        <ellipse rx="11" ry="4.2" />
+        <ellipse rx="11" ry="4.2" transform="rotate(60)" />
+        <ellipse rx="11" ry="4.2" transform="rotate(120)" />
+      </g>
+    </svg>
+  );
+}
